app/assets/javascripts/fae/form/_filtering.js

```diff
@@ -179,7 +179,10 @@ Fae.form.filtering = {
   _addSortingListeners: function() {
     var _this = this;
 
-    $('th[data-sort]').on('click', function() {
+    // Fizzle if listeneres have already been attached
+    if (this._are_sorting_listeners_setup === true) { return; }
+
+    $('body').on('click', 'th[data-sort]', function() {
```

Comment:
@aaronransley that is unusual. Educated guess is that Frob Core is being initialized before the DOM is ready, and therefore jQuery returns an empty array since it can't find the selector. There doesn't appear to be `FCH.$body` used elsewhere in the code, so ignore my comments. 

This should be an issue on [Frob Core](https://github.com/wearefine/frob-core). I'm surprised it hasn't come up in other projects.